lib/log.js

```diff
@@ -70,12 +72,14 @@ Lug.prototype.begin = function (op, request) {
   this.logger.debug(op)
 }
 
-Lug.prototype.event = function (name, data) {
+Lug.prototype.event = function (name, request, data) {
   var e = {
     event: name,
     data: unbuffer(data)
   }
-  process.stdout.write(JSON.stringify(e) + '\n')
+  data.metricsContext = metricsContext.add({},
```

Comment:
Indeed, nice catch - although I'm slightly confused that the tests were passing locally without this change